python/app.py

```
@@ -327,6 +325,11 @@ def process(game_id: int):
         response.headers["X-Result-Cache"] = "MISS"
         # _emit_metrics(timings, gameId, 200)
         return response, 200
+        try:
+            _emit_dq(game_id, game, processed_game)
+        except Exception as e:  # observability must never cost a render
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)